lib/util/version.js

```diff
@@ -14,8 +14,19 @@ function getFromContext(context) {
   return confVer.split('.').map(part => Number(part));
 }
 
-function test(context, methodVer) {
-  const confVer = getFromContext(context);
+function getFlowVersionFromContext(context) {
+  let confVer = '999.999.999';
+  // .eslintrc shared settings (http://eslint.org/docs/user-guide/configuring#adding-shared-settings)
+  if (context.settings.react && context.settings.react.flowVersion) {
+    confVer = context.settings.react.flowVersion;
+  } else {
+    throw 'Could not retrieve flowVersion from settings';
```

Comment:
We are guessing what the flowVersion is, but only for the specific implementation in the `prop-types` rule: https://github.com/yannickcr/eslint-plugin-react/pull/1377/files#diff-531a13f7edd3ed4e7c65c3a7df7052f0R868

As this is a general utlity function, I don't think that here we can easily guess what flow version we are using... I can move this utility function into the `prop-types` rule directly as it is currently the only place where is used. I just thought that for future cases it might be good to have it extracted already so that it can be used in other rules as well.